---
title: "Research"
permalink: /research/
author_profile: false
comments: false
layout: single
date: 2021-12-25
---

*Updated 2025.08.29*
## Current projects
A lot of my time I spend developing tools for genomic analyses of blood borne viruses such as Hepatitis C Virus. And currently I am developing a [Nextflow](https://www.nextflow.io/) pipeline for assembly, genotyping and antiviral resistance analysis of HCV called [HCVTyper](https://github.com/folkehelseinstituttet/hcv_illumina).  I am also doing research on the evolution and dispersal of HCV in Norway.  


## Previous projects  

### SARS-CoV-2  
I was hired at the [Norwegian Institute of Public Health](https://www.fhi.no/en/) during the Covid-19 pandemic and most of my time was spend tackling the vast amount of data that we produced from genome sequencing of SARS-CoV-2 in Norway. But I was also involved in a few research projects on SARS-CoV-2, such as [our study on the dynamics and evolution of the UK-lineage in Norway](/publications/Virus_evol_2021), or this on the [dynamics of the Norwegian AY.63 lineage](/publications/AY_63), or one study where we discovered a [novel Delta/Omicron recombinant](/publications/2024_SciRep).  

### Regulatory RNAs and the origin of animals
As a post doc I led two fully funded projects on genome regulation and the evolutionary origin of animals. We studied questions such as, How did the first animal look like, and how did it evolve from single-celled species? In particular. we were interested in the genes that *regulate* how other genes are used and how these may have contributed to the evolution of increased morphological and developmental complexity. We worked on sponges, comb jellies and the single-celled choanozoans, the closest ancestors of animals.

#### MicroRNAs
MicroRNAs are short RNA molecules that regulate other genes, and in animals microRNAs are an essential part of the genetic toolkit. I am trying to find out when microRNAs and the complementary protein machinery evolved in animals. We have recently discovered for the first time microRNAs, as well as homologs of the animal protein machinery which processes the miRNAs, in unicellular ancestors of animals, suggesting that these evolved much earlier than previously thought. The paper has been published in [Current Biology](https://www.cell.com/current-biology/fulltext/S0960-9822(18)31063-7#%20).

<figure style="width: 580px" class="align-center">
<img src="{{ site.url }}{{ site.baseurl }}/assets/images/research/Sphearoforma_Arctica_q22.jpg" alt="">
  <figcaption><i>Sphaeroforma arctica</i>, one of our unicellular ancestors. Photo: Jon Bråte.</figcaption>
</figure>

<figure style="width: 580px" class="align-center">
<img src="{{ site.url }}{{ site.baseurl }}/assets/images/publications/2018-miRNA-Figure1.png" alt="">
  <figcaption>Figure from the <i>Current Biology</i> paper showing the distribution of the Microprocessor proteins and miRNAs in animals and their single-celled relatives.</figcaption>
</figure>

And we have also extended these results to show that microRNAs are in fact present across Ichthyosporea (currently only available as a [preprint](https://jonbra.github.io/publications/2025_F1000)).  
  
<figure style="width: 580px" class="align-center">
<img src="{{ site.url }}{{ site.baseurl }}/assets/images/research/Ichthyo_miRNA_F1000.png" alt="">
  <figcaption>Figure from the <i>F1000Research</i> preprint showing an example of novel miRNAs in different species of Ichthyosporea.</figcaption>
</figure>


#### Long non-coding RNAs
Another type of regulatory genes are long non-coding RNAs. This is a class of recently discovered RNA genes. Very little is known about what these genes do, and even less is known about their evolutionary significance. To get a glimpse of how the last common ancestor of animals might have used lncRNAs, we have studied their expression dynamics during cellular development in the sponge _Sycon ciliatum_ ([published in _Proceedings B_](/publications/02-2015-proceedings/)) and in the ichthyosporean _Sphaeroforma arctica_ ([published in _eLife_](https://elifesciences.org/articles/49801)).  

<figure style="width: 580px" class="align-center">
<img src="{{ site.url }}{{ site.baseurl }}/assets/images/research/Sciliatum_insitu.jpg" alt="">
  <figcaption><i>in situ</i> hybridization of lncRNAs expressed in the oocytes of the sponge <i>Sycon ciliatum</i>. Photo: Jon Bråte.</figcaption>
</figure>

<figure style="width: 580px" class="align-center">
<img src="{{ site.url }}{{ site.baseurl }}/assets/images/research/Sarctica_lncRNA_heatmap.jpg" alt="">
  <figcaption> A heatmap of 70 lncRNAs with correlated expression patterns of coding genes during cellular development of <i>S. arctica</i>. Figure: Dudin <i> et al. </i> 2019. <i>bioRxiv</i>.</figcaption>
</figure>

### Protist diverst and evolution
I did my PhD on the diversity and evolution of protists, single-celled eukaryotes. In particular, I studied weird groups of protists that were largely unknown. I used methods such as environmental sequencing and molecular phylogenetics.
