---
title: Howe et al. 2022 JHEP Reports
layout: single
excerpt: "Title: Characteristics of hepatitis C virus resistance in an international cohort after a decade of direct-acting antivirals"
tags:
  - paper
  - HCV
  - paper_notes
categories:
  - paper_notes
---

**Title: Characteristics of hepatitis C virus resistance in an international cohort after a decade of direct-acting antivirals**  

Link to paper: [https://www.jhep-reports.eu/article/S2589-5559(22)00034-9/fulltext](https://www.jhep-reports.eu/article/S2589-5559(22)00034-9/fulltext)  

Resistance mutations against direct-acting antivirals (DAA) before and after treatment. 
Large dataset, 3355 patients from 22 countries (from the SHARED database). Treated with DAA. It's not entirely clear, but most likely they use consensus sequences from each patient. Hence, they will not catch any minority mutations present in the patients. I therefore think that the numbers of Resistance Associated Substitutions (RAS) is actually even higher than what they report. It is therefore not directly comparable to results from for example HCV GLUE run on bam files. 

They find that the frequency of RAS increases from baseline after exposure to DAA. From 37% to 60% for NS3, 29% to 80% for NS5A, 15% to 22% for NS5B for sofosbuvir and 24% to 37% in NS5B for dasabuvir. 

They divide their analyses per genotype (subtype) because the RAS are different for different genotypes. They also divide analyses per gene because not all sequences cover all genes.  

For NS3 and NS5B they only report if a sample had detecteble RAS or not. They do not report individual mutations. They perform the deepest mutational analyses for NS5A. Here they have good data for both before and after treatment. They did not have enough sequences for NS3 and NS5B to perform the analysis. But I wonder if they could have analysed the genotypes together? Or are the amino acid positions not comparable between genotypes?  

These are the reference sequences they used for the various genotypes:  
[Table S1](chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/https://www.jhep-reports.eu/cms/10.1016/j.jhepr.2022.100462/attachment/dd517d8a-be7c-4f9e-bfd7-5bd0e57a1691/mmc1.pdf):  
  
| Genotype | Reference strain | GenBank accession id | Length (bp) | Nucleotide position NS3-4A | Nucleotide position NS5A | Nucleotide position NS5B |
|----------|------------------|----------------------|-------------|----------------------------|--------------------------|--------------------------|
| GT 1a    | H77              | NC_004102            | 9646        | 3420-5474                  | 6258-7601                | 7602-9377                |
| GT 1b    | Con1             | AJ238799             | 9030        | 3420-5474                  | 6258-7598                | 7599-9374                |
| GT 2     | JFH-1            | AB047639             | 9678        | 3431-5485                  | 6269-7666                | 7667-9442                |
| GT 3     | S52              | GU814263             | 9555        | 3436-5490                  | 6274-7629                | 7630-9402                |
| GT 4     | ED43             | GU814265             | 9497        | 3419-5473                  | 6257-7591                | 7592-9364                |
| GT 5     | SA13             | AF064490             | 9408        | 3328-5382                  | 6166-7515                | 7516-9291                |
| GT 6     | EUHK2            | Y12083               | 9340        | 3374-5428                  | 6212-7564                | 7565-9340                |  





## Relevant papers and notes
[Notes on HCV RAS](/notes_on_HCV_RAS/)
