---
date: 2025-03-24
title: "Identification and Functional Characterization of microRNAs in Ichthyosporea"
permalink: /publications/2025_F1000
excerpt: "Identification of miRNAs in Ichthyosporea"
header:
  teaser: assets/images/publications/2025_F1000_teaser.png
gallery:
  - url: assets/images/publications/2025_F1000_Fig1.png
    image_path: assets/images/publications/2025_F1000_Fig1.png
    alt: "Figure 1. miRNA identification"
  - url: assets/images/publications/2025_F1000_Fig2.png
    image_path: assets/images/publications/2025_F1000_Fig2.png
    alt: "Figure 2. Length distribution of small RNA reads"
  - url: assets/images/publications/2025_F1000_Fig3.png
    image_path: assets/images/publications/2025_F1000_Fig3.png
    alt: "Figure 3. Examples of the identified ichthyosporean miRNAs"
  - url: assets/images/publications/2025_F1000_Fig4.png
    image_path: assets/images/publications/2025_F1000_Fig4.png
    alt: "Figure 4. Four groups of conserved miRNAs"
  - url: assets/images/publications/2025_F1000_Fig5.png
    image_path: assets/images/publications/2025_F1000_Fig5.png
    alt: "Figure 5. Category 0 targets identified by CleaveLand"
tags:
  - RNA
  - RNA-seq
  - miRNA
  - paper
---

Zhu, H. and **Br√•te, J**. 2025. *F1000Research*. [doi:https://doi.org/10.12688/f1000research.162501.1](https://doi.org/10.12688/f1000research.162501.1)  

{% include gallery caption="" %}


<h3>Abstract</h3>

**Background**  

microRNAs (miRNAs) are key post-transcriptional regulators of gene expression in animals, yet their origins and early functions remain elusive. In this study, we explore the diversity and functionality of miRNAs in Ichthyosporea, a group of unicellular organisms that represent some of the closest relatives of animals.

**Methods**

We have performed miRNA identification by integrating deep small RNA sequencing with draft genome assemblies across six species of Ichthyosporea, including _Abeoforma whisleri_, _Pirum gemmata_, and _Sphaeroforma_ sp. To assess the functional roles of these miRNAs, we have used computational target prediction and degradome sequencing.

**Results**

We identified a rich and varied repertoire of miRNAs across Ichthyosporea. Many of these miRNAs appear to be lineage-specific, while a subset is conserved among closely related species, suggesting both rapid evolutionary turnover and the persistence of ancient regulatory elements. Moreover, predicted target genes span a wide range of transcripts, implicating miRNAs in diverse cellular processes. Additionally, degradome sequencing reveal that in _S. arctica_, several miRNAs are likely to induce mRNA cleavage.

**Conclusions**

These findings not only extend previous reports of miRNA presence in Ichthyosporea but also underscore the deep evolutionary roots of miRNA-mediated gene regulation in the holozoan lineage. The miRNA induced target cleavage in _S. arctica_ is also observed in cnidarians and supports the hypothesis that mRNA cleavage may represent the ancestral function of animal miRNAs.

